CONTRIBUTING.md

```diff
@@ -610,6 +610,25 @@ Then put this:
 export NODE_OPTIONS=--no-experimental-fetch
 ```
 
+If Following type of error comes while running dev server(i.e using above commands):
+
+```bash
+Error: ENOSPC: System limit for number of file watchers reached
+```
+The meaning of this error is that the number of files monitored by the system has reached the limit!
+
+Do the following to Modify the number of system monitoring files :
+```bash
+sudo gedit /etc/sysctl.conf
```

Comment:
gedit may not be available on all operating systems. It is worth adding a comment saying `Use appropriate text editor for your OS`.